bucket/openmpt.json

```
@@ -0,0 +1,37 @@
+{
+    "homepage": "https://openmpt.org/",
+    "description": "OpenMPT is a powerful audio application that makes writing music fun, easy and efficient.",
+    "license": "BSD-3-Clause",
+    "version": "1.28.05.00",
+    "architecture": {
+        "32bit": {
+            "url": "https://openmpt.org/download_file.php?file=OpenMPT-1.28.05.00-Setup.exe",
```

I would rather use direct URL.

```suggestion
            "url": "https://download.openmpt.org/OpenMPT-1.28.05.00-Setup.exe",
```